src/core/Form/Textarea/Textarea.baseStyles.tsx

```diff
@@ -16,20 +16,12 @@ export const baseStyles = withSuomifiTheme(
       flex-direction: column;
```

Comment:
Should we have width for Textarea, like we are having for TextInput?

https://github.com/vrk-kpa/suomifi-ui-components/blob/de3ba041127f51eb2df5c75ff152933eefc16e13/src/core/Form/TextInput/TextInput.baseStyles.tsx#L7-L13

Reply:
That is question to ask from the Design Team I guess? Would be good to have something and when thinking about the component usage - one should be able to set the display and size attributes without breaking the component.

Reply:
This is asked from the design team, waiting for response.

Reply:
They said to use same default width as with TextInput, ` width: 290px;` if it does not cause any problems.